### DISCUSSION TITLE
How to use as a "true" wiki? - am I missing something?

### DISCUSSION THREAD

**Discussion by james-cook** [requarks/wiki#4256] [Help / Questions]
I am a long time user of wikis  (I met Ward Cunningham in 1996!) so I decided to try wiki.js today.
And it looks good :) But I am having a hard time using it as a wiki ;)

The problem is the speed (wiki) of page creation. In the original wiki you would just use CamelCase and save the page and click on that text, and there was your new page. In mediawiki you use double square brackets for a similar effect.

With wiki.js I am at a loss...
I tried: 
```
We would like to [submit an application](submit an application) by christmas
We would like to [submit an application]() by christmas
We would like to [](submit an application) by christmas
```
in the markdown editor - but clicking the results (where clickable) does not take me to a new page/page creation.

In the visual editor I have managed to create an "empty link" - containing e.g. the text "submit an application" (the link must have some text in it e.g. "submit_an_application2", an empty link will not work). Clicking this after saving the page does lead to a "Create Page" dialog were I am asked to name the page "again".

So the link text is different to the link address and this is different again the the actual page name...

I am probably missing something - is there any way to simply create a link over some text, e.g. "submit an application" and
by clicking on this link (after saving the page) have this text be used by default as the link address and the page name?

Thanks for any hints :)

**Comment by budinero** [comment]
Hi, it seems that links to pages are links to file paths. Therefore, the proper format would be:
`[submit an application](/page_folder/page_filename)`
The key here is to start with the slash `/`
